lib/instagram/response.rb

```diff
@@ -1,6 +1,8 @@
 module Instagram
   module Response
     def self.create( response_hash, ratelimit_hash )
+      return if handle_potential_error(response_hash)
```

Comment:
This is not the right place to handle this. Missing signature should raise a `403` and can be handled in `raise_http_exception.rb`. Please add tests for this too. Also I feel that `RequestMissingSignature` is a better name.

Reply:
I am no longer using this in production. Please feel free to make the requested changes and merge, otherwise this may end up an orphaned PR.